share/man/man7/networking.7

```diff
@@ -0,0 +1,59 @@
+.\"-
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.\" Copyright (c) 2023 Alexander Ziaee
+.\"
+.\" Redistribution and use in source and binary forms, with or without
+.\" modification, are permitted provided that the following conditions
+.\" are met:
+.\" 1. Redistributions of source code must retain the above copyright
+.\"    notice, this list of conditions and the following disclaimer.
+.\" 2. Redistributions in binary form must reproduce the above copyright
+.\"    notice, this list of conditions and the following disclaimer in the
+.\"    documentation and/or other materials provided with the distribution.
+.\"
+.\" THIS SOFTWARE IS PROVIDED BY THE AUTHORS AND CONTRIBUTORS ``AS IS'' AND
+.\" ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+.\" IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+.\" ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE
+.\" FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+.\" DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+.\" OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+.\" HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+.\" LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+.\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+.\" SUCH DAMAGE.
+.\"
+.Dd August 29, 2023
+.Dt "NETWORKING" 7
+.Os
+.Sh NAME
+.Nm networking ,
+.Nm wifi
+.Nd quickstart guide to connecting to a network
+.Sh EXAMPLES
+.Ss Connecting to an ethernet network with DHCP:
+.Dl # dhclient em0
+.Ss Connecting to a wifi network:
+Identify your wifi hardware:
+.Dl % sysctl net.wlan.devices
+Configure your wifi hardware as wlan0 interface:
+.Dl # sysrc wlans_iwn0="wlan0"
+Set that interface to negotiate a dhcp lease with wpa supplicant:
+.Dl # sysrc ifconfig_wlan0="WPA SYNCDHCP"
+Enter the wifi network's details:
+.Dl # echo "network={ssid="myssid" psk="mypsk"} >> wpa_supplicant.conf
+Restart the network interface daemon:
+.Dl # service netif restart
+.Ss Scanning for wifi networks:
+.Dl % ifconfig wlan0 scan
+.Ss Airplane mode:
+.Dl # service netif stop
+.Sh SEE ALSO
+.Xr bsdconfig 8 ,
+.Xr dhclient 8 ,
+.Xr ifconfig 8 ,
+.Xr wpa_supplicant 8
+.Pp
+.Fx
+handbook chapter 42: Advanced Networking
```

Comment:
```suggestion
The Advanced Networking chapter of the FreeBSD Handbook
```

The number is currently 34 (not 42), and may change.

42 is [reserved for future use](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#The_Answer_to_the_Ultimate_Question_of_Life,_the_Universe,_and_Everything_is_42).